std/algorithm/mutation.d

```diff
@@ -1219,9 +1211,22 @@ pure nothrow @safe @nogc unittest
     assert(s2.a == 42);
 }
 
-private void trustedMoveImpl(T)(ref T source, ref T target) @trusted
+// https://issues.dlang.org/show_bug.cgi?id=20869
+// `move` should propagate the attributes of `opPostMove`
+unittest
```

Comment:
Needs to be explicitly attributed no ?

Reply:
Don't think that attributes matter for this `unittest` as it doesn't call any function.

Reply:
Doesn't matter but the style checker demands it.

Reply:
Added `@system`